main/Main.hs

```diff
@@ -143,7 +150,8 @@ main = do
   -- Re-read the config on SIGUSR2, but only if we have a config file
```

Comment:
Now we would need to change that condition, right? If we start up without a config file, but with `PGRST_DB_URI`, do we still want to trigger a reload from the database?

Reply:
>  If we start up without a config file, but with PGRST_DB_URI, do we still want to trigger a reload from the database?

We also need the `PGRST_DB_ANON` and `PGRST_DB_SCHEMAS` for starting, but yes - supporting it is good for completeness. Though the common use case would be to reload with NOTIFY.